libc-top-half/musl/src/thread/pthread_create.c

```diff
@@ -502,7 +502,7 @@ int __pthread_create(pthread_t *restrict res, const pthread_attr_t *restrict att
 	new_tls_base -= (uintptr_t)new_tls_base & (tls_align - 1);
 
 	new = new_pthread;
-	new += pthread_align;
+	new = (void*)new + pthread_align;
```

Comment:
I think this changes the meaning of this line. Previously we were adding to a void* now we are adding to a pthread*. I don't really understand why we tread the alignment as sizeof pthread. But feel free to correct me on this.

Reply:
It's too early for me to read diffs. lgtm.